asmcli/components/ca/mesh-ca.sh

```diff
@@ -45,6 +45,10 @@ EOF
   if [[ "$ISTIOD_COUNT" -ne 0 ]]; then
     local REVISION; REVISION="$(retry 2 kubectl -n istio-system get pod -l app=istiod \
       -o jsonpath='{.items[].spec.containers[].env[?(@.name=="REVISION")].value}')"
+    local REV_NAME; REV_NAME="istio-${REVISION}"
+    if [[ "${REVISION}" = default ]]; then
+      REV_NAME="istio"
+    fi
     local RAW_TRUST_DOMAINS_ALIASES; RAW_TRUST_DOMAINS_ALIASES="$(retry 2 kubectl -n istio-system get configmap istio-"${REVISION}" \
```

Comment:
do we need to change this to use `kubectl -n istio-system get configmap "${REV_NAME}"`?

Reply:
Yes.